### ISSUE TITLE
creaminstaller not working.

### ISSUE THREAD

**Issue by Mabbe17** [pointfeev/CreamInstaller#263]
Creaminstaller dont work on my computer.  I am on win 11. I check everything and nothing I cant do.

**Comment by pointfeev** [comment]
First and foremost, note that the program currently only supports Windows 10+ 64-bit machines as seen here. If that does not apply to you, then make sure you've extracted the executable from the ZIP file before you've launched it, resolved your anti-virus, and have tried downloading the .NET Desktop Runtime mentioned under installation instructions above and restarting your computer. If none of the above work, then I simply cannot do anything about it, I do not control .NET. Either your system is not supported by the current version of .NET, or something is wrong/corrupted with your system.

**Comment by pointfeev** [comment]
Are you downloading it from the GitHub Releases page [here](https://github.com/pointfeev/CreamInstaller/releases/latest/download/CreamInstaller.zip)?

**Comment by Mabbe17** [comment]
Yes and I do what you says but nothing work. I deactivate my antivirus too. Its still not working.
